src/ecs/audio/systems/audio.rs

```diff
@@ -5,11 +5,12 @@ use std::sync::atomic::{AtomicBool, Ordering};
 use std::time::Duration;
 
 use cgmath::{Matrix4, Point3, Transform};
+use core::transform::Transform as TransformComponent;
 use rodio::{Sample, Source, SpatialSink};
 
 use ecs::{Entity, Fetch, Join, ReadStorage, System, WriteStorage};
 use ecs::audio::components::{AudioEmitter, AudioListener};
-use ecs::transform::Transform as TransformComponent;
+
```

Comment:
Formatter dying internally at this point.

Reply:
What's the problem here? The Transform component has moved to core, so it's now `core::transform::Transform` a few lines above ?

Reply:
It decided to have two spaces inside of one. It doesn't show well on the preview though.